include/zephyr/dt-bindings/pinctrl/silabs/siwx917-pinctrl.h

```diff
@@ -0,0 +1,538 @@
+#define TRACE_D0_PD6     SIWX917_GPIO(6, 0xFF, 18, 3, 6, 0)
+#define TRACE_D1_PA9     SIWX917_GPIO(13, 0xFF, 4, 0, 9, 0)
+#define TRACE_D1_PD1     SIWX917_GPIO(6, 0xFF, 13, 3, 1, 0)
+#define TRACE_D1_PD7     SIWX917_GPIO(6, 0xFF, 19, 3, 7, 0)
+#define TRACE_D2_PA10    SIWX917_GPIO(13, 0xFF, 5, 0, 10, 0)
+#define TRACE_D2_PD2     SIWX917_GPIO(6, 0xFF, 14, 3, 2, 0)
+#define TRACE_D2_PD8     SIWX917_GPIO(6, 0xFF, 20, 3, 8, 0)
+#define TRACE_D3_PA11    SIWX917_GPIO(13, 0xFF, 6, 0, 11, 0)
+#define TRACE_D3_PD3     SIWX917_GPIO(6, 0xFF, 15, 3, 3, 0)
+#define TRACE_D3_PD9     SIWX917_GPIO(6, 0xFF, 21, 3, 9, 0)
+
+#define UART2_CTS_PA11      SIWX917_GPIO(6, 0xFF, 6, 0, 11, 0)
+#define UART2_CTS_PC0       SIWX917_GPIO(12, 0xFF, 9, 2, 0, 0)
+#define UART2_CTS_PD3       SIWX917_GPIO(9, 0xFF, 15, 3, 3, 0)
+#define UART2_CTS_ULP1      SIWX917_GPIO(9, 6, 23, 4, 1, 1)
+#define UART2_CTS_ULP7      SIWX917_GPIO(6, 6, 29, 4, 7, 7)
+#define UART2_CTS_ULP9      SIWX917_GPIO(9, 6, 31, 4, 9, 9)
+#define UART2_RS485DE_PA9   SIWX917_GPIO(6, 0xFF, 4, 0, 9, 0)
+#define UART2_RS485DE_ULP11 SIWX917_GPIO(6, 6, 33, 4, 11, 11)
+#define UART2_RS485DE_ULP2  SIWX917_GPIO(6, 6, 24, 4, 2, 2)
+#define UART2_RS485EN_PA12  SIWX917_GPIO(6, 0xFF, 7, 0, 12, 0)
+#define UART2_RS485EN_PB10  SIWX917_GPIO(6, 0xFF, 0, 1, 10, 0)
+#define UART2_RS485EN_ULP0  SIWX917_GPIO(6, 6, 22, 4, 0, 0)
+#define UART2_RS485RE_PA8   SIWX917_GPIO(6, 0xFF, 3, 0, 8, 0)
+#define UART2_RS485RE_ULP1  SIWX917_GPIO(6, 6, 23, 4, 1, 1)
+#define UART2_RS485RE_ULP10 SIWX917_GPIO(6, 6, 32, 4, 10, 10)
+#define UART2_RTS_PA10      SIWX917_GPIO(6, 0xFF, 5, 0, 10, 0)
+#define UART2_RTS_PB11      SIWX917_GPIO(6, 0xFF, 0, 1, 11, 0)
+#define UART2_RTS_PB12      SIWX917_GPIO(6, 0xFF, 0, 1, 12, 0)
+#define UART2_RTS_PB15      SIWX917_GPIO(12, 0xFF, 9, 1, 15, 0)
+#define UART2_RTS_PD2       SIWX917_GPIO(9, 0xFF, 14, 3, 2, 0)
+#define UART2_RTS_ULP0      SIWX917_GPIO(9, 6, 22, 4, 0, 0)
+#define UART2_RTS_ULP6      SIWX917_GPIO(6, 6, 28, 4, 6, 6)
+#define UART2_RTS_ULP8      SIWX917_GPIO(9, 6, 30, 4, 8, 8)
+#define UART2_RX_PA6        SIWX917_GPIO(6, 0xFF, 1, 0, 6, 0)
+#define UART2_RX_PB13       SIWX917_GPIO(6, 0xFF, 0, 1, 13, 0)
+#define UART2_RX_PC1        SIWX917_GPIO(12, 0xFF, 9, 2, 1, 0)
+#define UART2_RX_ULP10      SIWX917_GPIO(9, 6, 32, 4, 10, 10)
+#define UART2_RX_ULP2       SIWX917_GPIO(9, 6, 24, 4, 1, 1)
+#define UART2_RX_ULP4       SIWX917_GPIO(6, 6, 26, 4, 4, 4)
+#define UART2_RX_ULP8       SIWX917_GPIO(6, 6, 30, 4, 8, 8)
+#define UART2_TX_PA15       SIWX917_GPIO(2, 0xFF, 8, 0, 15, 0)
+#define UART2_TX_PA7        SIWX917_GPIO(6, 0xFF, 2, 0, 7, 0)
+#define UART2_TX_PB14       SIWX917_GPIO(6, 0xFF, 0, 1, 14, 0)
+#define UART2_TX_PC2        SIWX917_GPIO(12, 0xFF, 9, 2, 2, 0)
+#define UART2_TX_ULP11      SIWX917_GPIO(9, 6, 33, 4, 11, 11)
+#define UART2_TX_ULP3       SIWX917_GPIO(9, 6, 25, 4, 1, 1)
+#define UART2_TX_ULP5       SIWX917_GPIO(6, 6, 27, 4, 5, 5)
+#define UART2_TX_ULP9       SIWX917_GPIO(6, 6, 31, 4, 9, 9)
```

Comment:
Any way to align the value on the comma? I would be easier to identify every argument:
```
#define UART2_RS485RE_ULP10 SIWX917_GPIO( 6,    6, 32, 4, 10, 10)
#define UART2_RTS_PA10      SIWX917_GPIO( 6, 0xFF,  5, 0, 10,  0)
#define UART2_RTS_PB11      SIWX917_GPIO( 6, 0xFF,  0, 1, 11,  0)
#define UART2_RTS_PB12      SIWX917_GPIO( 6, 0xFF,  0, 1, 12,  0)
#define UART2_RTS_PB15      SIWX917_GPIO(12, 0xFF,  9, 1, 15,  0)
#define UART2_RTS_PD2       SIWX917_GPIO( 9, 0xFF, 14, 3,  2,  0)
#define UART2_RTS_ULP0      SIWX917_GPIO( 9,    6, 22, 4,  0,  0)
#define UART2_RTS_ULP6      SIWX917_GPIO( 6,    6, 28, 4,  6,  6)
#define UART2_RTS_ULP8      SIWX917_GPIO( 9,    6, 30, 4,  8,  8)
#define UART2_RX_PA6        SIWX917_GPIO( 6, 0xFF,  1, 0,  6,  0)
#define UART2_RX_PB13       SIWX917_GPIO( 6, 0xFF,  0, 1, 13,  0)
#define UART2_RX_PC1        SIWX917_GPIO(12, 0xFF,  9, 2,  1,  0)
#define UART2_RX_ULP10      SIWX917_GPIO( 9,    6, 32, 4, 10, 10)
#define UART2_RX_ULP2       SIWX917_GPIO( 9,    6, 24, 4,  1,  1)
#define UART2_RX_ULP4       SIWX917_GPIO( 6,    6, 26, 4,  4,  4)
#define UART2_RX_ULP8       SIWX917_GPIO( 6,    6, 30, 4,  8,  8)
#define UART2_TX_PA15       SIWX917_GPIO( 2, 0xFF,  8, 0, 15,  0)
#define UART2_TX_PA7        SIWX917_GPIO( 6, 0xFF,  2, 0,  7,  0)
#define UART2_TX_PB14       SIWX917_GPIO( 6, 0xFF,  0, 1, 14,  0)
#define UART2_TX_PC2        SIWX917_GPIO(12, 0xFF,  9, 2,  2,  0)
#define UART2_TX_ULP11      SIWX917_GPIO( 9,    6, 33, 4, 11, 11)
#define UART2_TX_ULP3       SIWX917_GPIO( 9,    6, 25, 4,  1,  1)
#define UART2_TX_ULP5       SIWX917_GPIO( 6,    6, 27, 4,  5,  5)
#define UART2_TX_ULP9       SIWX917_GPIO( 6,    6, 31, 4,  9,  9)
...
```